internal/pkg/manifest/validate.go

```diff
@@ -924,6 +951,17 @@ func (v vpcConfig) Validate() error {
 	return nil
 }
 
+// Validate returns nil if rdwsPlacement is configured correctly.
+func (p rdwsPlacement) Validate() error {
+	if err := (Placement)(p).Validate(); err != nil {
+		return err
+	}
+	if string(p) == string(PublicSubnetPlacement) {
+		return fmt.Errorf(`public is not supported for %s`, RequestDrivenWebServiceType)
+	}
+	return nil
+}
```

Comment:
what do you think about flipping this conditional instead? 
```go
if string(p) == string(PrivateSubnetPlacement) {
   return nil
}
return fmt.Errorf("placement: '%s' is not supported for %s", string(p), RequestDrivenWebServiceType)
```
This way in case we add more placement options we won't have to update this method 🤔